### ISSUE TITLE
Default styler fills LineStrings

### ISSUE THREAD

**Issue by kdpenner** [SciTools/cartopy#1782]
### Description

Default styler fills LineStrings. IMO this behavior is unexpected; I expect LineStrings to be plotted as line strings. The workaround is to set `edgecolor` and `facecolor`.

#### Code to reproduce

```python
from shapely.geometry import LineString
import cartopy.crs as ccrs
import matplotlib.pyplot as plt


geoms = [LineString([(-40, 10), (-30, 15), (-30, 20)]),
         LineString([(-40, -10), (-30, -15), (-30, -20)])]

robin = ccrs.Robinson()

fig = plt.figure(figsize=(10, 7))
ax = fig.add_subplot(1, 1, 1, projection=robin)
ax.add_geometries(geoms, crs=ccrs.PlateCarree())
ax.coastlines()
plt.show()
```

![t](https://user-images.githubusercontent.com/9297904/117577010-fe7b6180-b0b5-11eb-93d6-fe0ea21f9121.png)

**Comment by greglucas** [comment]
I agree with you, I thought there were some other issues mentioning this brought up a while ago, but I can't find them now. It doesn't seem like it would be too hard to change the defaults around with an `isinstance(LineString)` check when adding features.
